spel/scripts/amigen7-build.sh

```diff
@@ -29,6 +29,7 @@ EPELRELEASE="${SPEL_EPELRELEASE:-https://dl.fedoraproject.org/pub/epel/epel-rele
 EPELREPO="${SPEL_EPELREPO:-epel}"
 EXTRARPMS="${SPEL_EXTRARPMS}"
 FIPSDISABLE="${SPEL_FIPSDISABLE}"
+GRUBTMOUT="${SPEL_GRUBTMOUT:-5}" ; export GRUBTMOUT
```

Comment:
Instead of an export, I think we should be passing this through the `--grub-timeout` option when calling `GrubSetup.sh`

Reply:
This seemed like the least-coding-required method (one line here rather than adding a line here and modifying the subsequent script-invocation in the respective `amigen-{7,8}.sh` scripts.

Reply:
Yeah, we've just switched most everything else already over to using args instead of envs. We can certainly take a separate task to modify the GrubSetup.sh call to use args though.

Reply:
The AMIgen scripts that set up GRUB each accept flagged-args for overriding the default `GRUB_TIMEOUT` setting. It was mostly just the "add one line" vs "add a line and modify a line" thing.

/shug

I'll update the spel script-invokers.

Reply:
Resolved in 5588f7d